### PR TITLE
Enable Export E2E Test

### PR DESCRIPTION
**Pull request by wsugarman**
## Description
- Enable Export E2E Test in PaaS
- Minor refactor to remove unnecessary async behavior
- Refactor some docker usage

## Related issues
N/A

## Testing
Existing tests

### CODE REVIEW

File: build/pr/run-e2e-tests.yml [hunk 1]
```diff
@@ -14,7 +14,10 @@ jobs:
     workingDirectory: 'docker'
 
   - bash: for i in {1..12}; do curl -fsS "$(testEnvironmentUrl)health/check" > /dev/null && exit 0 || sleep 5; done; exit 1
-    displayName: 'Wait for the DICOM Server to Start'
+    displayName: 'Wait for DICOM Server'
+
+  - bash: for i in {1..12}; do curl -fsS "$(testFunctionsUrl)" > /dev/null && exit 0 || sleep 5; done; exit 1
```
Comment (jnlycklama) [inline build/pr/run-e2e-tests.yml:19]: What does the base functions url give us? Some status on the functions like our /health/check endpoint?

Comment (wsugarman) [reply build/pr/run-e2e-tests.yml:19]: This is actually the URL that may optionally host a web page that says something like "your function app is ready to go." However, I am working separately on a health check endpoint for our Azure functions. The PR is actually in draft, but it's not ready yet: https://github.com/microsoft/dicom-server/pull/1739

Comment (wsugarman) [reply build/pr/run-e2e-tests.yml:19]: However, for our own local testing, I want to make sure that at least the Function Host started successfully before attempting any tests.

Comment (jnlycklama) [reply build/pr/run-e2e-tests.yml:19]: Ok makes sense! Thanks for the info

**Review by jnlycklama — APPROVED** [review]

**Pull request merged**